docs/Software Architecture Document/Components View/index.md

```diff
@@ -102,15 +101,32 @@ The view is represented by module and subsystem diagrams that show the system's
   - *Constraints*
     - *N/A* 
 
+#### RPC Service
+  - *Responcibility*
+    - Send Mobile/HMI RPCs
+    - Handle Mobile/HMI RPCs
+  - *Relations*
+    - Uses ***PluginManager***
+    - Uses ***RequestController***
+    - Requires ***ProtocolHandler** and **ProtocolObserver (Protocol Handler)***
+    - Requires ***HMIMessageObserver** and **HMIMessageSender (HMI Message Handler)***
+  - *Interfaces* 
+    - Provides ***RPCService*** interface
+  - *Behavior*
+    - The component implements RPC transfering functionality
+  - *Constraints*
+    - *N/A* 
+
 #### Commands
```

Comment:
where does the Commands component go to in the diagram? Seems it is no longer there.